google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionRunner.java

```diff
@@ -73,4 +73,6 @@
    * {@link #run(TransactionCallable)} has returned normally.
    */
   Timestamp getCommitTimestamp();
+
+  TransactionRunner allowNestedTransaction();
```

Comment:
add detailed documentation for this method, including the information that this is not good practice. only for legit cases (eg: txn across DBs ).

provide examples of nesting which we would like to discourage